### PR TITLE
Feature/support multiple platforms podspec

### PR DESCRIPTION
**Pull request by jaelim-ms**

### CODE REVIEW

File: MobileCenter.podspec [hunk 1]
```diff
@@ -16,59 +16,66 @@ Pod::Spec.new do |s|
                       Mobile Center Distribute will let your users install a new version of the app when you distribute it via the Mobile Center. With a new version of the app available, the SDK will present an update dialog to the users to either download or postpone the new version. Once they choose to update, the SDK will start to update your application. This feature will NOT work if your app is deployed to the app store.
 
                       3. Mobile Center Distribute:
-                      Mobile Center Distribute provides the capability to display in-app updates to your app users when a new version of the application is released.
+                      Mobile Center Distribute provides the capability to display in-app updates to your app users when a new version of the application is released. Not available for macOS SDK.
```
Comment (clpolet) [inline MobileCenter.podspec]: I am wondering if there is a way to make the "MacOS unavailability" more visible?

File: MobileCenter.podspec [hunk 2]
```diff
@@ -16,59 +16,66 @@ Pod::Spec.new do |s|
...
-      ss.vendored_frameworks = "MobileCenter-SDK-iOS/MobileCenter.framework"
+    ss.frameworks = 'Foundation', 'SystemConfiguration'
+    ss.ios.frameworks = 'CoreTelephony', 'UIKit'
+    ss.osx.frameworks = 'AppKit'
+    ss.ios.vendored_frameworks = "MobileCenter-SDK-Apple/iOS/MobileCenter.framework"
+    ss.osx.vendored_frameworks = "MobileCenter-SDK-Apple/macOS/MobileCenter.framework"
   end
 
  s.subspec 'Analytics' do |ss|
-      ss.frameworks = 'Foundation', 'UIKit'
```
Comment (clpolet) [inline MobileCenter.podspec]: Maybe revise the indentation here.

Comment (jaelim-ms) [inline MobileCenter.podspec]: Fixed.

File: MobileCenter.podspec (hunk 1 above)
Comment (jaelim-ms) [inline MobileCenter.podspec]: That's a good idea but looks description doesn't have any formatting functionalities.
What about

>1. Mobile Center Analytics (iOS and macOS):
>Mobile Center Analytics helps you understand user behavior and customer engagement to improve your app. The SDK automatically captures session count, device properties like model, OS version, etc. You can define your own custom events to measure things that matter to you. All the information captured is available in the Mobile Center portal for you to analyze the data.
>2. Mobile Center Crashes (iOS and macOS):
>Mobile Center Distribute will let your users install a new version of the app when you distribute it via the Mobile Center. With a new version of the app available, the SDK will present an update dialog to the users to either download or postpone the new version. Once they choose to update, the SDK will start to update your application. This feature will NOT work if your app is deployed to the app store.
>3. Mobile Center Distribute (iOS only):
>Mobile Center Distribute provides the capability to display in-app updates to your app users when a new version of the application is released. Not available for macOS SDK.
>4. Mobile Center Push (iOS only):
>Mobile Center Push enables you to send push notifications to users of your app from the Mobile Center portal. Not available for macOS SDK.

**Pull request merged**